dist/index.js

```diff
@@ -7,12 +7,14 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
 var isArray = require("lodash/isArray");
+var isBoolean = require("lodash/isBoolean");
 var isEqual = require("lodash/isEqual");
 var isEmpty = require("lodash/isEmpty");
 var isNil = require("lodash/isNil");
 var isNumber = require("lodash/isNumber");
 var isObject = require("lodash/isObject");
 var set = require("lodash/set");
+var get = require("lodash/get");
```

Comment:
i forget that its not react, so i was confused at first why you set these to a var. but i get it now.